### DISCUSSION TITLE
Error edit in angular

### DISCUSSION THREAD

**Discussion by negrinlinda2** [geoman-io/leaflet-geoman#1247] [Questions & Help]
In angular 12 the edition does not work correctly.
I have the latest version of geoman. The rest of the options do work. You can see in the image the wrong vertex markers. Also when finished it returns to the original geometry.
Does anyone have an idea how it can be fixed?
Packages:
"leaflet": "^1.9.2",
  "@geoman-io/leaflet-geoman-free": "^2.13.0"

I've tried various settings and nothing.
![Captura](https://user-images.githubusercontent.com/2333399/198321460-5fd2c3df-545a-4981-aece-a62b025a6018.PNG)

**Comment by Falke-Design** [comment]
Can you please create a demo. You can use one of this projects as base: https://codesandbox.io/search?query=geoman%20angualar&page=1&configure%5BhitsPerPage%5D=12
